src/actions/ApplyAcl.ts

```diff
@@ -31,8 +31,10 @@ import { ERROR_KIND_FATAL, ERROR_KIND_PERMISSION } from "../ErrorCache";
  * @param {Mjolnir} mjolnir The Mjolnir client to apply the ACLs with.
  */
 export async function applyServerAcls(lists: BanList[], roomIds: string[], mjolnir: Mjolnir): Promise<RoomUpdateError[]> {
+    const serverName: string = new UserID(await config.RUNTIME.client!.getUserId()).domain;
```

Comment:
I suspect that many modules will need that, so might as well make it a property `mjolnir.serverName`.